dev_tools/scripts/todo_bicg.txt

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
This file should not be committed, it's was only generated to help going through the implementation process.